packages/notifications/lib/firebase.js

```diff
@@ -31,6 +34,10 @@ if (!FIREBASE_PROJECT_ID || !FIREBASE_CLIENT_EMAIL || !FIREBASE_PRIVATE_KEY || !
 }
 
 const publish = async (args) => {
+  if (SEND_NOTIFICATIONS === 'false' || (DEV && SEND_NOTIFICATIONS !== 'true')) {
```

Comment:
I don't agree:
- I expect the default behavior to be: don't send notifications in development (and thus not require the APN/Firebase key setup, just to try the code), but send them in production (without the danger of notifications not working because I forgot to add a stupid boolean config var).
- That is also consistent with the behavior of `SEND_MAILS`: https://github.com/orbiting/backends/blob/3b20d6975f0ff5ec8d0847bbf8ba80a5c1c17876/packages/mail/lib/sendMailTemplate.js#L45

Reply:
> I expect the default behavior to be: don't send notifications in development (and thus not require the APN/Firebase key setup, just to try the code), but send them in production (without the danger of notifications not working because I forgot to add a stupid boolean config var).

Got it. Would have aimed to not send any notification if flag is not explicitly set but see very much why that is unhealthy in production.